src/features/bulk/FileViewer/FileViewer.tsx

```diff
@@ -0,0 +1,66 @@
+import React, { useCallback, useEffect, useState } from 'react';
+import { DocumentViewer } from '@amsterdam/bmi-component-library';
+import debounce from 'debounce';
+
+import { MetadataGenericType } from '../../../types';
+import { useAppSelector } from '../../hooks';
+import { getChangeIndividualFields, getDefaultFields, getFields } from '../bulk/store/selectors';
+import { IBulkFile } from '../bulk/store/model';
+import { TGetDocumentViewUrl } from '../bulk/types';
+
+import DefaultFieldsTable from './DefaultFieldsTable';
+import IndividualFieldsForm from './IndividualFieldsForm';
+import {
+	FileViewerDocumentInnerStyle,
+	FileViewerDocumentStyle,
+	FileViewerFieldsStyle,
+	FileViewerStyle,
+} from './styles';
+import { DEFAULT_DEBOUNCE } from '../bulk/constants';
```

Comment:
Constants in typescript work best as an enum IMO. This way you have automatic type safety and protection from typos.

Reply:
I've typed `DEFAULT_DEBOUNCE` to a number, so it has a better type safety what way.
@fortunato Could you explain how this enum solution would work and what the benefits would be? I don't understand how it would protest from typos better.